projects/library/paginator/package.json

```diff
@@ -4,7 +4,8 @@
       "entryFile": "src/public-api.ts",
       "umdModuleIds": {
         "@terminus/ngx-tools": "terminus.ngxTools",
-        "@terminus/ngx-tools/coercion": "terminus.ngxTools.coercion"
+        "@terminus/ngx-tools/coercion": "terminus.ngxTools.coercion",
+        "@terminus/ngx-tools/utilities": "terminus.ngxTools.utilities"
```

Comment:
Wonder how it worked if `utilities` is needed but not included in the package?

Reply:
Without it you'll see this warning during build:

![Screen Shot 2020-03-30 at 7 48 53 AM](https://user-images.githubusercontent.com/270193/77909204-f0280880-725a-11ea-9f3f-10a4a0b977b8.png)

And it usually guesses wrong since our modules are scoped. I haven't personally verified, but I'd expect this would affect consumers in certain scenarios.